### ISSUE TITLE
importlib_resources and Ubuntu 20.04

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#6084]
Ubuntu 20.04 has [Python 3.8](https://packages.ubuntu.com/focal/python3) but apparently [not `importlib_resources`](https://packages.ubuntu.com/search?keywords=importlib-resources&searchon=names).

Maybe we should add a fallback to the older (stdlib) API or `pkg_resources` in that case? See the discussion with @eli-schwartz here: https://github.com/qutebrowser/qutebrowser/issues/4467#issuecomment-740060813

Alternatively, we could just force those users to install in a virtualenv instead, where they can easily install it via pip (or use the older qutebrowser apt package). Will need to think about it some more.

**Comment by eli-schwartz** [comment]
It is not possible to fall back on the older (stdlib) API as far as I know, since the only way I could see to implement it was using the API added in 3.9... 

Falling back to pkg_resources is possible but seems entirely awkward. the Requires-Dist metadata would still demand that importlib_resources be installed, and you cannot tell it "on python 3.8, depend on importlib_resources except on Ubuntu, where depend on setuptools instead".
- You could require neither, but try importing both... but that's terrible UX.
- You could require both, but that doesn't solve the problem at all.
- You could require importlib_resources, but tell Ubuntu users or distro maintainers "you may patch setup.py to depend on pkg_resources instead, and the code contains fallbacks just for you", but that's... still kind of terrible UX.

Keeping things as they are:
- If Ubuntu wishes to ship qutebrowser, it is their responsibility to ship the dependencies too.
- If users on Ubuntu wish to `pip install qutebrowser`, it will ***resolve dependencies*** and install importlib_resources to the same place it installs qutebrowser.

tl;dr I see no problem here.

**Comment by The-Compiler** [comment]
I was mainly thinking about people running from the repository directly, without running `setup.py`. But yeah, the only reason to run against system-packages in that scenario is for proprietary Qt codecs - and given that our `mkvenv.py` [supports symlinking a system-wide PyQt](https://github.com/qutebrowser/qutebrowser/blob/master/doc/install.asciidoc#installing-qutebrowser-with-virtualenv), I don't think this is worth the trouble.
